src/insecure/plaintext.js

```diff
@@ -0,0 +1,88 @@
+'use strict'
+
+const protobuf = require('protons')
+const protocol = '/plaintext/2.0.0'
```

Comment:
Can you put this const after all `require`? I prefer to have all the `require` first and then constants